MetadataExtractor/MetadataExtractor.csproj

```diff
@@ -15,6 +15,7 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageId>MetadataExtractor</PackageId>
     <PackageTags>Metadata;Exif;IPTC;XMP;ICC;Photoshop;WebP;PNG;BMP;ICO;PCX;JPEG;TIFF;PSD;Photography;QuickTime;MOV;MP4;M4V;Video</PackageTags>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
```

Comment:
I think this change should be in `Directory.Build.props` rather than here.

https://github.com/drewnoakes/metadata-extractor-dotnet/blob/0fe00658a7b06732de8b65128899f1716783f211/Directory.Build.props#L12

Reply:
You are probably right, although it still must be PackageLicenseExpression.

Reply:
Yes it should be `PackageLicenseExpression`. It's not project specific though, so should be higher up.

Reply:
Agreed; closing.